scripts/zones/RuLude_Gardens/npcs/Harith.lua

```diff
@@ -53,7 +53,7 @@ entity.onTrigger = function(player, npc)
     if
         player:getQuestStatus(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.EMPTY_MEMORIES) == QUEST_AVAILABLE and
         (player:getCurrentMission(xi.mission.log_id.COP) >= xi.mission.id.cop.THE_MOTHERCRYSTALS or
-         player:getCharVar("Mission[6][128]Option") > 0)
+         player:getCharVar("Mission[6][118]Option") > 0)
```

Comment:
hold on this check would check the PREVIOUS mission before the_mothercrystals

Reply:
yes, as you don't get the mothercrystals until you are deep into your first promy. (And animas are available as soon as you first enter promyvion)